pkg/bot-code/put_all_simple_strings_into_consts.go

```diff
@@ -0,0 +1,143 @@
+// Package constants provides centralized string constants for the frankmeza-anthropic-bot project
```

Comment:
refactor: 

- make consts for all strings in `bot_ai`, `bot_blog`, `bot_code`, `bot_github`,
- move them into file `pkg/constants/index.go`